### ISSUE TITLE
ERC20 assets on Polygon not displayed

### ISSUE THREAD

**Issue by Naxsun** [tallycash/extension#2060]
### Discord Discussion Link

na

### What browsers are you seeing the problem on?

Chrome

### What were you trying to do?

View wallet balance

### What did not work?

On an account I had already imported in the wallet, before the wallet was updated from 0.14.6 --> 0.14.7, the ERC20 assets on mainnet are showing, but the assets on Polygon aren't. Just Matic.

<img width="372" alt="Screen Shot 2022-08-29 at 19 21 01" src="https://user-images.githubusercontent.com/71384922/187259537-d9054e14-8e6b-4d25-8988-48415f21f6b1.png">

When importing a new account, the Polygon ERC20 assets were displayed for the new address. For the old address it remained just Matic.

Logs:
[Uploading logs_v0_14_7__2022-08-29T19_19_44+02_00.txt…]()

Already imported account before update: 0x77f6e4ea32f0b807dcaaddbec9dabe0131614cd5

Newly imported account: 0x208e94d5661a73360d9387d3ca169e5c130090cd

### Version

Community Edition v0.14.7

**Comment by Naxsun** [comment]
Didn't allow me to add the logs above.

[logs_v0_14_7__2022-08-29T19_19_44+02_00.txt](https://github.com/tallycash/extension/files/9446858/logs_v0_14_7__2022-08-29T19_19_44%2B02_00.txt)

**Comment by 0xDaedalus** [comment]
This line is interesting:

```
"Alchemy token balance response didn't validate, did the API change?,{"address":"[REDACTED] reverted"}},{"contractAddress":"[REDACTED] have property 'tokenBalance'"}]"
```

**Comment by mhluongo** [comment]
Another API change eh? :dizzy_face:

**Comment by mhluongo** [comment]
What tokens are you expecting here @Naxsun? I see DAI and some others auto-discovered on Polygon locally

**Comment by mhluongo** [comment]
Hmmm, I bet this is because of the hotfix + our having a spotty token list for Polygin :thinking:

**Comment by mhluongo** [comment]
Closed by #2067
